testsuite/smoke-test/src/rosetta.rs

```diff
@@ -477,6 +511,23 @@ async fn create_staking_contract(
     info.client().submit_and_wait(&txn).await.unwrap()
 }
 
+async fn unlock_stake(
+    info: &AptosPublicInfo<'_>,
+    account: &mut LocalAccount,
+    operator: AccountAddress,
+    amount: u64,
+) -> Response<Transaction> {
+    let unlock_stake = info
+        .transaction_factory()
+        .payload(aptos_stdlib::staking_contract_unlock_stake(
+            operator, amount,
+        ))
+        .sequence_number(2);
```

Comment:
This shouldn't be hardcoded, as it'll be easy to break.  Could you pass it into this function?